api/v1alpha1/rediscachefirewallrule_types.go

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
+// NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
+
+// RedisCacheFirewallRuleSpec defines the desired state of RedisCacheFirewallRule
+type RedisCacheFirewallRuleSpec struct {
+	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
+	// Important: Run "make" to regenerate code after modifying this file
+
+	ResourceGroup string                           `json:"resourceGroup"`
+	CacheName     string                           `json:"redisCache,omitempty"`
```

Comment:
Since we required the rediscache name to add a firewallrule, i would drop 'omitempty' for this field for rediscache

Reply:
Ahh yes! Very true! I changed the IP addresses/properties to drop `omitempty` because they are also required :) Thanks!!